Dockerfile

```diff
@@ -36,6 +36,11 @@ RUN rm irods-icommands.deb
 # Install test coverage module
 RUN pip install coverage
 
+# Install haystack and sphinx
+RUN pip install django-haystack
+RUN pip install sphinx

```

Comment:
Yes, the `pip install sphinx` call is for the document generator per request of Alva as he's used it in other projects and wanted it available here as well.